gcp-intro/python/README.md

```diff
@@ -0,0 +1,40 @@
+---
+author: Josh Kodroff
+language: python
+provider: google
+use: both
+last-ran: none
+boilerplate: none
+last-host: none
+learn-content: none
+original-repo: none
+status: active
+deck: unknown
+---
+
+# Google Cloud Platform with Pulumi Workshop
+
+This workshops guides you through using Pulumi's [Google Cloud (GCP) Classic](https://www.pulumi.com/registry/packages/gcp/) provider.
+
+## Pre-requisites
+
+### Tools
+
+You will need the following to complete this workshop:
+
+* A [Pulumi account and token](https://www.pulumi.com/docs/intro/pulumi-service/accounts/#access-tokens?utm_source=da&utm_medium=referral&utm_campaign=workshops&utm_content=gcp-intro-java)
+  * If you don't have an account, go to the [signup page](https://app.pulumi.com/signup?utm_source=da&utm_medium=referral&utm_campaign=workshops&utm_content=gcp-intro-java).
+* The [Pulumi CLI](https://www.pulumi.com/docs/get-started/install/?utm_source=da&utm_medium=referral&utm_campaign=workshops&utm_content=gcp-intro-java)
```

Comment:
```suggestion
* A [Pulumi account and token](https://www.pulumi.com/docs/intro/pulumi-service/accounts/#access-tokens?utm_source=da&utm_medium=referral&utm_campaign=workshops&utm_content=gcp-intro-python)
  * If you don't have an account, go to the [signup page](https://app.pulumi.com/signup?utm_source=da&utm_medium=referral&utm_campaign=workshops&utm_content=gcp-intro-python).
* The [Pulumi CLI](https://www.pulumi.com/docs/get-started/install/?utm_source=da&utm_medium=referral&utm_campaign=workshops&utm_content=gcp-intro-python)
```

Reply:
Again, my bad. Thank you for careful review!